### DISCUSSION TITLE
BT-777-Lot : no schematron rules

### DISCUSSION THREAD

**Discussion by jcagnolo** [OP-TED/eForms-SDK#843] [Q&A] [answered]
Hello, 

Our developers are noticing a strange behavior concerning the BT-777-Lot field:
- There are no schematron rules for this field. 
- For the rule to appear, you need to fill in the BT-06-notice field. 

Is this normal? Have you observed this behavior? Do you know if this behavior will be corrected later? 

Thank you 
JC

**Comment by YvesJo** [comment, marked as answer]
HI,
BT-06 is optional (cf Regulation Annex). The description of a Strategic Procurement is mandatory and only allowed when it is about a Strategic Procurement. Rule will get active as part of the progressive rules activation with SDK 1.11

**Comment by jcagnolo** [comment]
Thank you for your reply. 
I'll pass it on to our developers and get back to you if I need any clarification.
